### PR TITLE
fixed compiler and include locations

### PR DESCRIPTION
**Pull request by DylanSchneider**
Story: https://jira.logrhythm.com/browse/NM-1104

### CODE REVIEW

**Review by alexweltman — CHANGES_REQUESTED** [review]
wow Dylan, I expected better out of AEPi

File: scripts/CodeCoverage.py [hunk 1]
```diff
@@ -76,15 +76,15 @@ def get_cmake():
...
     if PROBE_BUILD:
-        default_args += ' -Wl,-rpath -Wl,. -Wl,-rpath -Wl,/usr/local/probe/lib -Wl,-rpath -Wl,/usr/local/probe/lib64 '
+        default_args += ' -Wl,-rpath -Wl,. -Wl,-rpath -Wl,/usr/local/gcc/lib -Wl,-rpath -Wl,/usr/local/gcc/lib64 '
     default_args += '\''
     return default_args
 
 def get_compiler():
     if PROBE_BUILD:
-        return '-DCMAKE_CXX_COMPILER=/usr/local/probe/bin/g++'
+        return '-DCMAKE_CXX_COMPILER=/usr/local/gcc/bin/g++'
```
Comment (KjellKod) [inline scripts/CodeCoverage.py:87]: I don't think this is correct.  Shouldn't it be `/usr/bin/g++`?

Comment (KjellKod) [reply scripts/CodeCoverage.py:87]: I stand corrected!  from the package files I see that it is `/usr/local/gcc/bin`

File: scripts/CodeCoverage.py [hunk 2]
```diff
@@ -133,7 +133,7 @@ def get_gcovr():
 
 def get_gcov():
     if PROBE_BUILD:
-        return '--gcov-executable /usr/local/probe/bin/gcov'
+        return '--gcov-executable /usr/bin/gcov'
```
Comment (KjellKod) [inline scripts/CodeCoverage.py]: This would point to gcov for version `4.8.5` while the compiler in `/usr/local/gcc/bin` points to a newer version of gcc

Comment (KjellKod) [reply scripts/CodeCoverage.py]: I.e. shouldn it stay `/usr/local/gcc/bin/gcov`?

**Comment by KjellKod** [comment]
👍

**Review by KjellKod — APPROVED** [review]
nice change

**Pull request merged**